egs2/lrs3/avsr1/README.md

```diff
@@ -0,0 +1,29 @@
+<!-- Generated by scripts/utils/show_asr_result.sh -->
+# RESULTS
+## Environments
+- date: `Thu Sep 28 23:59:06 KST 2023`
+- python version: `3.8.18 (default, Sep 11 2023, 13:40:15)  [GCC 11.2.0]`
+- espnet version: `espnet 202308`
+- pytorch version: `pytorch 1.12.0`
+- Git hash: `5d0758e2a7063b82d1f10a8ac2de98eb6cf8a352`
+  - Commit date: `Wed Aug 30 18:03:42 2023 -0400`
+- Pretrained model: [https://huggingface.co/espnet/msk_lrs3_train_avsr_avhubert_large_extracted_en_bpe1000](https://huggingface.co/espnet/msk_lrs3_train_avsr_avhubert_large_extracted_en_bpe1000)
+
+## exp/asr_train_avsr_avhubert_large_extracted_en_bpe1000
+### WER
+
+|dataset|Snt|Wrd|Corr|Sub|Del|Ins|Err|S.Err|
+|---|---|---|---|---|---|---|---|---|
+|inference_asr_model_valid.acc.ave/test|1321|9890|98.5|1.1|0.4|0.2|1.7|8.8|
```

Comment:
It is very good.
Is it an expected number?
The performance of this task is already saturated?